src/calculateRentalCost.js

```diff
@@ -3,8 +3,28 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days >= 7) {
+    totalCost -= 50;
```

Comment:
Avoid using literals in expressions. Use constants to explain what the value `50` represents, as per the checklist item 2. For example, you could define `const LONG_TERM_DISCOUNT = 50;` and use it in the expression.